chainer/functions/__init__.py

```diff
@@ -226,6 +226,8 @@
 from chainer.functions.math.prod import prod  # NOQA
 from chainer.functions.math.prod import Prod  # NOQA
 from chainer.functions.math.scale import scale  # NOQA
+from chainer.functions.math.sign import sign  # NOQA
+from chainer.functions.math.sign import Sign  # NOQA
```

Comment:
Please remove `Sign`. We're not going to expose new function classes.

Reply:
Fixed.